packages/matchbox/src/components/Toggle/Toggle.js

```diff
@@ -56,6 +56,7 @@ class Toggle extends Component {
           onFocus={onFocus}
           onBlur={onBlur}
           type='checkbox'
+          data-id="toggle-input"
```

Comment:
We don't want to add it directly to the component, we want to pass it in as a prop. I'll show you in the next file